zdm/misc_functions.py

```diff
@@ -28,6 +28,190 @@
 from zdm import parameters
 
 
+def get_w_tau_dist(grid,norm=True):
```

Comment:
I might recommend that all `w` and `tau` methods go into their own `w_tau.py` module.  Or even one for each! :)

Reply:
This is a to-do, to see how these pan out